website/content/docs/release-notes/1.12.0.mdx

```diff
@@ -0,0 +1,136 @@
+
+Some of these enhancements and changes in this release include the following:
+
+
+- Vault Enterprise now supports PKCS#11 provider plugin (client library) functionality.
+- Vault Enterprise can now manage keys for Oracle TDE. This requires the Advanced Data Protection license.
+- PKI Key revocation improvements are made to Vault’s PKI engine, introducing a new OCSP responder and automatic CRL rebuilding (with up-to-date Delta CRL), that offers significant performance and data transfer improvements to revocation workflows.
+- BYOK in Transform engines now allow users to import their keys generated elsewhere.
+- KMIP Server Profile adds support for additional operations, allowing  Vault to claim support for the baseline server profile.
+- Transform secrets engine supports time-based auto-key rotation for tokenization.
+- Path and Role-based Quotas extend the existing Vault Quota support by allowing quotas to be extended to the API path suffixes and auth mount roles.
+- Licensing termination behavior has changed where non-evaluation licenses (production licenses) will no longer have a termination date.
+- TFVP includes improvements made to the Vault Version Detection and Auth engine.
+
+## New Features
+
+This section describes the new features introduced in Vault 1.12.0.
+
+##3 Transform Secrets Engine Enhancements
+
+These features need the Vault Enterprise ADP License.
+
+### Bring Your Own Key(BYOK) for Transform
+
+In release 1.11, we introduced BYOK support to Vault, enabling  customers to import existing keys into the Vault Transit Secrets Engine and  enabling secure and flexible Vault deployments.
+We  are extending that support to the Vault Transform Secrets Engine in this release.
+
+### MSSQL Support
+
+An MSSQL store is now available to be used as an external storage engine with tokenization Transform Secrets Engine. Refer to the following documents, [Transform Secrets Engine(API)](/api-docs/secret/transform), [Transform Secrets Engine](/docs/secrets/transform/index), and [Tokenization Transform](/docs/secrets/transform/tokenization) for more information.
+
+### Key Auto Rotation
+
+Periodic rotation of encryption keys is a recommended key management practice for a good security posture. In Vault release 1.10, we added support for Auto key rotation for Transit Secrets Engine. In Vault 1.12, the Transform secrets engine is now  enhanced, allowing users to set the rotation policy during key creation in a time interval, which will cause Vault to rotate the Transform keys when the time interval elapses automatically.
+Refer to the following documents, [Transit Secrets Engine](/docs/secrets/transit) and [Transit Secrets Engine(API)](/api-docs/secret/transit#rotate-key) for more information.
```

Comment:
@schultz-is / @rculpepper -- should this point to Transform rotate-key docs perhaps?

Reply:
ah yes, good catch. key auto-rotation functionality was added for transform in this release

Reply:
@schultz-is , hi. Please provide the doc link and I'll replace the existing link(s). Thanks.

Reply:
```suggestion
Refer to the following documents, [Tokenization Transform](/docs/secrets/transform/tokenization) and [Transform Secrets Engine(API)](/api-docs/secret/transform#rotate-tokenization-key) for more information.
```

@taoism4504 sure thing! these will probably cover it